Source/WebCore/platform/network/curl/CurlContext.cpp

```diff
@@ -342,10 +341,12 @@ void CurlHandle::enableSSLForHost(const String& host)
         setSslVerifyHost(CurlHandle::VerifyHost::StrictNameCheck);
     }
 
-    const auto& cipherList = sslHandle.getCipherList();
-    if (!cipherList.isEmpty())
+    if (const auto& cipherList = sslHandle.cipherList(); !cipherList.isEmpty())
```

Comment:
This is checking `cipherList.isEmpty()`. But, the default value of CURLOPT_SSL_CIPHER_LIST is `NULL`. So, I think it's OK to set emtpy string ("").
I don't think we need a local variable `cipherList` here because it's a simple getter.

Reply:
`NULL` means to use OpenSSL's default value. On the other hand, an empty string means that no cipher is set. These values have distinctly different meanings.

Also, the description for CURLOPT_SSL_CIPHER_LIST mentions "one or more cipher strings".
https://curl.se/libcurl/c/CURLOPT_SSL_CIPHER_LIST.html
```
The list must be syntactically correct, it consists of one or more cipher strings separated by colons.
```

Reply:
CString::data() returns `nullptr` if it doesn't set any data. So I delete a local variable `cipherList`.